sdk/core/Azure.Core/samples/Serialization.md

```diff
@@ -30,18 +30,29 @@ Deserialization
 
 ## Using explicit cast
 
+When using protocol methods for advanced handling of RequestContext it is still possible to use the strongly typed models.
+There is an explicit cast operator that can be used to convert the protocol Response to the strongly typed model.
+There is also an explicit cast operator that can be used to convert the strongly typed model to the protocol RequestContent.
+
 Serialization
 
 ```C# Snippet:ExplicitCast_Serialize
-//TODO
+PetStoreClient client = new PetStoreClient(new Uri("http://somewhere.com"), new MockCredential());
+DogListProperty dog = new DogListProperty("myPet");
+Response response = client.CreatePet("myPet", (RequestContent)dog);
```

Comment:
If we've generated a model but only have a protocol method, could we possibly make this "just work" with an implicit cast?

Reply:
https://github.com/Azure/azure-sdk-for-net/issues/36394